app/views.py

```diff
@@ -33,3 +41,25 @@ def page_forbidden(error):
 @url.errorhandler(500)
 def page_server_error(error):
     return render_template('500.html', title='Server Error'), 500
+
+
+def convert_to_readable_time(time):
```

Comment:
all this stuff (evreything from this line and below) is not needed in this PR AFAICT